components/tooltip/README.md

```diff
@@ -7,20 +7,30 @@ Tooltips display additional information when users focus or hover on a point of
 <script type="module">
   import '@brightspace-ui/core/components/button/button.js';
   import '@brightspace-ui/core/components/tooltip/tooltip.js';
+  import '@brightspace-ui/core/components/tooltip/tooltip-help.js';
 
   window.addEventListener('load', function () {
     setTimeout(function() {
       var tooltip = document.querySelector('#tooltip');
+      var helpTooltip = document.querySelector('#help-tooltip').shadowRoot.querySelector('d2l-tooltip');
```

Comment:
We should be using `const` instead of `var`, even in README examples.

Reply:
Yeah I agree, I thought it a bit odd that the existing code for that was using `var`. Assumed perhaps there was a compatibility reason or something (needed to use ES5 etc). 

Tried it out right now with `const`  locally and everything seems to still work fine, so updating this.